csrankings-q.csv

```diff
@@ -11,7 +11,8 @@ Qi Li 0012,Iowa State University,https://www.cs.iastate.edu/people/qi-li,NOSCHOL
 Qi Liu 0003,USTC,http://staff.ustc.edu.cn/~qiliuql,5EoHAFwAAAAJ
 Qi Qi 0001,BUPT,https://scs.bupt.edu.cn/info/1299/2737.htm,NOSCHOLARPAGE
 Qi Rose Yu,Univ. of California - San Diego,http://roseyu.com,4HTITaMAAAAJ
-Qi Sun 0002,New York University,http://qisun.me,oN7gaqMAAAAJ
+Qi Sun 0002,Zhejiang University,http://qisunchn.top,4OnODOAAAAAJ

```

Comment:
Not in CS. Please justify how they meet the inclusion criteria. Publishing in particular conferences is not justification.

Reply:
Qi SUN @ Zhejiang University received **the Ph.D. degree from the Dept. of Computer Science & Engineering, Chinese University of Hong Kong**, under the supervision of Prof. Bei Yu (http://www.cse.cuhk.edu.hk/~byu) in Jul. 2022. Note that Prof. Bei Yu ranks first in the EDA track in CSRanking. 

Zhejiang University established the School of Integrated Circuits this year, and all teachers in the EDA field were moved to this school. That's why Prof. Qi SUN is affiliated with the School of Integrated Circuits. For example, Prof. Cheng ZHUO @ Zhejiang University is now affiliated with the School of Integrated Circuits and is classified as an EDA researcher at your CSRanking.

Qi Sun's research interests include **ML in EDA**, design space exploration, etc. His work has been recognized with the **2021 ICCAD Best Paper Award**, the **Bronze Medal of the 2021 ICCAD Student Research Competition**, and a **Best Paper Award Nomination of DATE**.

**He has published 45 papers, among which 33 papers are EDA papers (DAC/ICCAD/DATE/ASPDAC/TCAD/TODAES, please refer to https://qisunchn.top/publication.html)**. Currently, his research directions include design for manufacturability and simulation optimization. Both are typical and important EDA topics.

Reply:
This does not speak to the inclusion requirements.